src/main/scala/dpla/ingestion3/enrichments/EnrichmentDriver.scala

```diff
@@ -2,10 +2,25 @@ package dpla.ingestion3.enrichments
 
 import dpla.ingestion3.model._
 
+/**
+  * TODO: assign the hostname from a config file or commandline switch?
+  *
+  * @see dpla.ingestion3.enrichments.Twofisher
+  * @see SpatialEnrichmentIntegrationTest
+  */
+object Geocoder extends Twofisher {
+  override def hostname = {
+    System.getenv("GEOCODER_HOST") match {
```

Comment:
At some point, we're going to need to figure out a better way to inject configuration. It also will have to deal with a distributed setup.